azavea/summer-of-maps-website
# Update development environment to Ubuntu 16.04

## Overview

This updates the base Python version beyond Python 2.6.9, which contains
support for SNI (now necessary for HTTPS interactions with Ansible
Galaxy). In addition, it updates all of the project dependencies too:

- Docker
- Docker Compose
- Jekyll
- Vagrant
- Ansible

Fixes https://github.com/azavea/summer-of-maps-website/issues/45
Fixes https://github.com/azavea/summer-of-maps-website/issues/48

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- First, destroy any instances of the development environment
- Use `setup` to bring up a new development environment
- Use `server` within the virtual machine to see interactive changes locally
